node/utils/LicenseManager.ts

```diff
@@ -81,6 +82,24 @@ export class LMClient extends ExternalClient {
     )
   }
 
+  public checkUserAdminPermission = async (
+    account: string,
+    userEmail: string,
+    resourceCode: string
+  ) => {
+    const productCode = B2B_LM_PRODUCT_CODE // resource name on lincense manager = B2B
+
+    const checkOrgPermission = await this.get<boolean>(
```

Comment:
How is the response time of this request? It is important to know what the impact of this request is. 

What happens if the request fails due to a timeout or response error? The expected is to block all requests?